pkg/metrics/metrics.go

```diff
@@ -726,8 +728,6 @@ func ResetMetricsForClusterManagers() {
 	clusterStatusCountMetric.Reset()
 	kafkaPerClusterCountMetric.Reset()
 	clusterStatusCapacityMaxMetric.Reset()
-	clusterStatusCapacityUsedMetric.Reset()
-	clusterStatusCapacityAvailableMetric.Reset()
```

Comment:
These three metrics could probably be calculated in one reconciler. 
There will be some work around these metrics for dynamic scaling tasks. We could use the opportunity to revisit them and calculate them in one reconciler.